src/aspire/basis/steerable.py

```diff
@@ -132,10 +142,77 @@ def calculate_bispectrum(
 
         return B
 
-    def rotate(self, complex_coef, radians, refl=None):
+    def rotate(self, coef, radians, refl=None):
+        """
+        Returns coefs rotated by `radians`.
+
+        :param coef: Basis coefs.
+        :param radians: Rotation in radians.
+        :param refl: Optional reflect image (bool)
+        :return: rotated coefs.
+        """
+        # Covert radians to a broadcastable shape
+        if isinstance(radians, np.ndarray):
+            if len(radians) != len(coef):
+                raise RuntimeError(
+                    "`rotate` call `radians` length cannot broadcast with"
+                    f" `coef` {len(coef)} != {len(radians)}"
+                )
+            radians = radians.reshape(-1, 1)
+        # else: radians can be a constant
+
+        # self.angular_indices are `ks`
+        assert len(self.angular_indices) == coef.shape[-1]
+
+        # For all coef in stack,
+        #   compute the ks * radian used in the trig functions
+        ks_rad = np.atleast_2d(self.angular_indices * -1 * radians)
```

Comment:
I like precomputing this but we can take out the minus sign and adjust the formula starting at 190 to be more intuitive (and match my pdf 😄)

Reply:
(intuitive in the sense of not multiplying by -1 with no explanation)

Reply:
in all fairness I now know what the explanation is (multiplying `radians` to be negative) but I'm also ambivalent about that